foo.html

```diff
@@ -0,0 +1 @@
+Hi
```

Comment:
It would be better if we change it to hello world

Reply:
Ok. I'll change it

Reply:
Done